lib/asciidoctor-bibtex/processor.rb

```diff
@@ -48,13 +48,17 @@ def apply(value)
   # current document, and run the different steps to add the citations
   # and bibliography
   class Processor
-    def initialize(bibfile, links = false, style = 'ieee', locale = 'en-US',
+    def initialize(bibfiles, links = false, style = 'ieee', locale = 'en-US',
```

Comment:
For backwards compatibility it would be better to accept both a single string and an array of strings here. This would also simplify the patch in the tests since single-bib could still be handled with the old syntax, and only the new test using an array would have to be introduced.

Reply:
String arguments restored in latest patch series

Also restored the old tests to pass strings instead.